2-virtual-machines-containers/2.1-golden-images.md

```diff
@@ -5,14 +5,35 @@
 ![](img2/linux_light.svg ':size=100x100 :class=light-mode-icon')
 ![](img2/linux_dark.svg ':size=100x100 :class=dark-mode-icon')
 
+?> If you have an m1 mac checkout virtualbox beta build for arm64. ( Note that as of 23/02/2023 the virtualbox beta build for arm64 did not work)
+
+?> assuming you have an m1 mac & the virtualbox beta is still not operational, the only m1 compatible section is the compatibility section.
+
+### M1 Compatibility Excersize
+
+1. You will need to install the utm project. assuming you have brew  `brew install --cask utm` should work
+
+2. You will also need an ARM64 linux iso; ideally we want to use the centos9 iso located [here](https://mirrors.centos.org/mirrorlist?path=/9-stream/BaseOS/aarch64/iso/CentOS-Stream-9-latest-aarch64-dvd1.iso&redirect=1&protocol=https) make sure to select the ARM64 version.
```

Comment:
The directions make it seem like the user will have to find and select the right iso, but we directly link an iso. Mirror links like this are often break. I think we should just link back to https://www.centos.org/download/ as that will likely not be changing and if it does I imagine they will handle redirecting for a while.

Reply:
Additionally you link to the full blown centOS download. I think we could probably get away with the minimal image

Reply:
I see why you just linked the iso now. This changes from centOS Linux to CentOS Stream. And it appears that on the Stream releases there is only a single version. Kind of crummy to have to download a 7GB file for this exercise. That fact alone is making me think we just target CentOS Linux 7 but with the EOL next year I could see this going either way.

Reply:
Well in testing I ran into issues booting a CentOS 7 minimal iso in UTM so Stream is fine with me

Reply:
I do think it is worth adding an info section to advise users to change the software option in the GUI install to be 'server'. Correct me if I am wrong but I think the default options will give you a GUI experience of CentOS and I imagine we are more likely to be dealing with GUI-less versions with clients (or I hope so)

Reply:
this should be fixed in https://github.com/liatrio/devops-bootcamp/pull/232/commits/a7d01112b7035bc6782caf367e6a7c486cde8787